subsys/pm/pm.c

```diff
@@ -153,7 +154,8 @@ bool pm_system_suspend(int32_t kernel_ticks)
 	 * We need to find out first the ticks to the next event
 	 */
 	events_ticks = pm_policy_next_event_ticks();
-	ticks = ticks_expiring_sooner(kernel_ticks, events_ticks);
+	ticks = (events_ticks != -1) ?
```

Comment:
This seems unrelated to the other changes?

Reply:
removed .